### PR TITLE
[ASM] - EXPANDR-1610 - Qualys integration in ASM pack

### PR DESCRIPTION
**Pull request by capanw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/EXPANDR-1610

## Description
Enriching remote IP using Qualys information.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml [hunk 1]
```diff
@@ -885,6 +983,4 @@ outputs:
   type: unknown
 tests:
 - No tests (auto formatted)
-fromversion: 6.8.0
-contentitemexportablefields:
-  contentitemfields: {}
+fromversion: 6.5.0
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml:986]:
```suggestion
fromversion: 6.8.0
```

File: Packs/CortexAttackSurfaceManagement/pack_metadata.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "Cortex Attack Surface Management",
     "description": "Content for working with Attack Surface Management (ASM).",
     "support": "xsoar",
-    "currentVersion": "1.5.9",
+    "currentVersion": "1.6.1",
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/pack_metadata.json:5]:
```suggestion
    "currentVersion": "1.5.10",
```
We haven't talked about what is revision vs minor version, but typically have been doing minor versions

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_1.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_1.md:1]: Would need to change the name of this file to reference the new version number

File: README.md [hunk 4]
```diff
@@ -10,41 +10,50 @@
 If you already have VS Code and Docker installed, you can click the badge above or [here](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=git@github.com:demisto/content.git) to get started. Clicking these links opens VS Code, installs the `Remote-Containers` extension (if not installed), clones the source code into a container volume, and spins up a development container, configured with all recommended settings.
```
Comment (johnnywilkes) [inline README.md:10]: What is this README and why is it in your change?

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
@capanw , looking good!  Few minor changes

File: Packs/CortexAttackSurfaceManagement/pack_metadata.json [hunk 5]
```diff
@@ -54,6 +54,10 @@
         "Tenable_io": {
             "mandatory": false,
             "display_name": "Tenable.io"
+        },
+        "qualys": {
+            "mandatory": true,
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/pack_metadata.json:59]:
```suggestion
            "mandatory": false,
```

**Comment by johnnywilkes** [comment]
@capanw , looks like for this error: https://app.circleci.com/pipelines/github/demisto/content/245929/workflows/16375176-809b-422a-9ea7-4f133ad00791/jobs/693378
We need to add Xpanse tag on Qualys https://github.com/demisto/content/blob/c4e0b2a514c3bc8ebcb32198b1090fdefad6f775/Packs/qualys/pack_metadata.json#L16 pack.  I would add this into this PR

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml (hunk 1 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml:986]: https://app.circleci.com/pipelines/github/demisto/content/245929/workflows/16375176-809b-422a-9ea7-4f133ad00791/jobs/693378?invite=true#step-119-1145
 
this change will fix this

**Comment by dansterenson** [comment]
@efelmandar please go over the new playbooks and playbook changes

File: README.md (hunk 4 above)
Comment (capanw) [reply README.md:10]: This is not intended. I think something to do with VS code. I will remove this.

Comment (BigEasyJ) [reply README.md:10]: Version control gremlins must have gotten in again.

File: Packs/CortexAttackSurfaceManagement/README.md [hunk 6]
```diff
@@ -76,12 +79,14 @@ The main active response playbook is the `Cortex ASM - ASM Alert` playbook. This
   - [Cortex ASM - SNMP Check](#cortex-asm---snmp-check)
   - [Cortex ASM - Splunk Enrichment](#cortex-asm---splunk-enrichment)
   - [Cortex ASM - Tenable.io Enrichment](#cortex-asm---tenableio-enrichment)
+  - [Cortex ASM - Qualys Enrichment](#cortex-asm---qualys-enrichment)
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/README.md:82]: Please make this alphabetical.

Please move this line up and below line 73 to look like:
```
- [Cortex ASM - GCP Enrichment](#cortex-asm---gcp-enrichment)
- [Cortex ASM - Qualys Enrichment](#cortex-asm---qualys-enrichment)
- [Cortex ASM - Rapid7 Enrichment](#cortex-asm---rapid7-enrichment)
```

File: Packs/CortexAttackSurfaceManagement/README.md [hunk 7]
```diff
@@ -172,6 +177,12 @@ Playbook that given the IP address enriches Tenable.io information relevant to A
 
 ![Cortex ASM - Tenable.io Enrichment](https://raw.githubusercontent.com/demisto/content/master/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Tenable.io_Enrichment.png)
 
+#### Cortex ASM - Qualys Enrichment
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/README.md:180]: Please make this alphabetical and move it up and under the `Cortex ASM - GCP Enrichment` section

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment_README.md [hunk 8]
```diff
@@ -1,41 +1,50 @@
 Used as a container folder for all enrichments of ASM alerts.
 
 ## Dependencies
+
 This playbook uses the following sub-playbooks, integrations, and scripts.
 
 ### Sub-playbooks
+
 * Cortex ASM - GCP Enrichment
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment_README.md:9]: **Non-blocking / Nit**: if you don't mind please make this alphabetical order as well.

**Review by BigEasyJ — CHANGES_REQUESTED** [review]

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml (hunk 1 above)
Comment (capanw) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml:986]: Thanks for the suggestions. Should have the new changes in the next commit.

File: Packs/CortexAttackSurfaceManagement/README.md (hunk 7 above)
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/README.md]: Addressed in f3c6bce. Thanks @capanw!

File: Packs/CortexAttackSurfaceManagement/README.md (hunk 6 above)
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/README.md]: Addressed in f3c6bce. Thanks @capanw!

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml (hunk 1 above)
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml]: Addressed in f3c6bce. Thanks @capanw!

File: Packs/qualys/pack_metadata.json [hunk 9]
```diff
@@ -15,6 +15,7 @@
     "keywords": [],
     "marketplaces": [
         "xsoar",
-        "marketplacev2"
+        "marketplacev2",
+        "Xpanse"
```
Comment (BigEasyJ) [inline Packs/qualys/pack_metadata.json]:
```suggestion
        "xpanse"
```

Comment (BigEasyJ) [reply Packs/qualys/pack_metadata.json]: This should fix the [validation error](https://app.circleci.com/pipelines/github/demisto/content/246085/workflows/2e4cd2df-0675-477d-a9f0-1207b9bdce22/jobs/693516) and [unit test / linting errors](https://app.circleci.com/pipelines/github/demisto/content/246085/workflows/2e4cd2df-0675-477d-a9f0-1207b9bdce22/jobs/693516)

File: README.md (hunk 4 above)
Comment (capanw) [reply README.md]: Undid those changes. It was just reformatting spaces.

**Review by BigEasyJ — APPROVED** [review]

**Comment by johnnywilkes** [comment]
@capanw , can we change version to 1.6.2 because https://github.com/demisto/content/pull/25482 has been approved

**Comment by johnnywilkes** [comment]
@dansterenson , please hold off on merge until we change version # to 1.6.2

**Review by dansterenson — APPROVED** [review]
Nice

**Pull request merged**